setup.py

```diff
@@ -58,7 +58,15 @@ def main():
               "dagrt>=2019.1",
               "grudge>=2015.1",
               "six>=1.8",
-              ])
+              "importlib-resources>=1.1.0; python_version < '3.9'",
+              ],
+
+          include_package_data=True,
+          package_data={
```

Comment:
IMO `mechanisms/__init__.py` shouldn't be necessary. There's not an `__init__.py` in pyopencl's package data either.

Reply:
51f3378

Reply:
To be continued

Reply:
I'm afraid I've gotten even deeper into `__init__.py`.  Is my use of it still in error? (0b6cb49)